documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,47 +1,73 @@
 == Raspberry Pi Touch Display
 
-The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. In recent versions of Raspberry Pi OS, both the HDMI and LCD display outputs can be used at the same time.
+The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. While the panel is connected, you can use both it, and the the normal HDMI display output at the same time.
 
-=== Board Support
+The Touch Display will function with all models of Raspberry Pi. Although, the earliest Raspberry Pi models, which lack appropriate mounting holes, require additional mounting hardware to fit the stand-offs on the display PCB.
 
-The DSI display is designed to work with all models of Raspberry Pi, however early models that do not have mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
+You can mount a Raspberry Pi to the back of the Touch Display using its stand-offs and then connect the appropriate cables between each device, depending on your use case. You can also mount the Touch Display in a separate chassis if you have one available. The connections remain the same, though you may need longer cables depending on the chassis you use.
```

Comment:
"appropriate"? There are in-appropriate cables? What cables?

Reply:
Used it in the sense to imply different cables. Data and power cables would be appropriate, in this case. Will reword though.

Reply:
Forgot to reword it in the latest commits, but still plan on doing so. There was just...a lot to get through.

Reply:
Perhaps "appropriate" -> "necessary"? :man_shrugging: